### ISSUE TITLE
networkmanager_initrc_domtrans fails to use new script labels

### ISSUE THREAD

**Issue by sshambar** [fedora-selinux/selinux-policy#1238]
In policy 36+, the interface networkmanager_initrc_domtrans (contrib/networkmanager.if) is still documented to:

"Execute NetworkManager scripts with an automatic domain transition to initrc"

Which it does for NetworkManager_initrc_exec_t labeled scripts.

However, since NM scripts have been relabeled from NetworkManager_initrc_exec_t to NetworkManager_dispatcher_script_t - the current interface no longer works.

NOTE: even if NetworkManager_dispatcher_script_t is permitted to transition to initrc_t, all the "templated"  scripts (dhclient, chronyc, etc) would be needed to permit the transition too to retain the current behavior.

**Comment by zpytela** [comment]
I am not sure what is the issue here. The interface still exists and works, just the dispatcher plugins transition to a different domain.

**Comment by sshambar** [comment]
Used as: `networkmanager_initrc_domtrans(mydomain_t)`

Old behavior: cmd running as mydomain_t - runs NM script (labeled NetworkManager_initrc_exec_t)  -> script runs as initrc_t

New behavior: cmd running as mydomain_t - runs NM script (now labeled NetworkManager_dispatcher_script_t) -> SELinux denies execute permissions on script.

**Comment by zpytela** [comment]
@sshambar I am not sure what is the problem here. Do you have a custom policy for mydomain?

As you already know, the labels have changed and there are subsequent changes in the policy for NetworkManager dispatcher scripts.

**Comment by sshambar** [comment]
@zpytela Yes, I was trying to create a policy for a domain that needs to execute NM scripts... however, the current `networkmanager_initrc_domtrans()` interface only includes the following at contrib/networkmanager.if:127

`init_labeled_script_domtrans($1, NetworkManager_initrc_exec_t)`

As the NM scripts are now labeled NetworkManager_dispatcher_script_t, this does not actually allow execution of the scripts...
